app/javascript/shared/components/focustrap.jsx

```diff
@@ -0,0 +1,30 @@
+import PropTypes from 'prop-types';
+import { h, Fragment } from 'preact';
+import { useEffect, useRef } from 'preact/hooks';
+import { createFocusTrap } from 'focus-trap';
```

Comment:
I added this library https://bundlephobia.com/result?p=focus-trap@6.1.2  It's size is **2.3kb** minified and gzipped is this ok?